src/gwt/src/org/rstudio/studio/client/server/remote/RemoteServer.java

```diff
@@ -1508,6 +1508,18 @@ public void showVignette(String topic, String pkgName)
                   params,
                   null);
    }
+
+   public void followHelpTopic(String topic, String pkg, ServerRequestCallback<JsArrayString> requestCallback)
+   {
+      JSONArray params = new JSONArray();
+      params.set(0, new JSONString(topic));
```

Comment:
If 'topic' or 'pkg' could ever be null, you might need to check that first. This is part of the motivation for the JSONArrayBuilder class, which does that check here:

https://github.com/rstudio/rstudio/blob/c256feb77f5151e0ba60409884ec28761f6e9c8d/src/gwt/src/org/rstudio/studio/client/common/JSONArrayBuilder.java#L90-L95

Reply:
Thanks. At the moment they cannot, because this only gets used based on matching this regex. `"^.*/help/library/([^/]*)/help/(.*)$"` but I've added for future proofing.